FWCore/Concurrency/interface/SerialTaskQueue.h

```diff
@@ -209,8 +210,9 @@ namespace edm {
 
   template <typename T>
   tbb::task* SerialTaskQueue::QueuedTask<T>::execute() {
-    try {
+    CMS_SA_ALLOW try {
       this->m_action();
+      // Caught exception is rethrown further below.
```

Comment:
@gartung Could you move the comment above the `CMS_SA_ALLOW`, i.e.
```cpp
    // Caught exception is rethrown further below.
    CMS_SA_ALLOW try {
```
since the comment is more for the `CMS_SA_ALLOW` than the `try` (in other places as well)?

Could you also remove the empty comments? In the end they may raise more questions/confusion than they "cure" the formatting.